cmd/jujud-controller/agent/model/manifolds_test.go

```diff
@@ -419,15 +442,15 @@ var expectedIAASModelManifoldsWithDependencies = map[string][]string{
 
 	"async-charm-downloader": {
 		"agent",
-		"api-caller",
```

Comment:
I was anticipating more of these would disappear as transient. I guess it will materialise as more controller-side workers go direct via services.

Reply:
Yeah, I wasn't happy about that either.

There is one dependency that everyone relies on, and that requires an api-caller. Need to track it down.